aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -47,6 +48,34 @@
                 { preventDefault: function () {} })
             ).toEqual(false);
         });
+
+        it('Cl.events._handler returns undefined if direction is "next" and month is 7', function () {
```

Comment:
function returns undefined if there's a direction, everything else is not really needed
what you should test instead is that the $.ajax was called with correct parameters given different input